packages/core/src/typeschema/validation.test.ts

```diff
@@ -790,16 +790,6 @@ describe('Legacy tests for parity checking', () => {
     }
   });
 
-  test('StructureDefinition', () => {
```

Comment:
I believe this test is identical to the test on line 318

Reply:
It is, great catch!